modules/dnn/src/onnx/onnx_graph_simplifier.cpp

```diff
@@ -190,19 +196,23 @@ class ExtractScalesSubgraph : public Subgraph
     {
         opencv_onnx::NodeProto* constant_node = inputs[1].dynamicCast<ONNXNodeWrapper>()->node;
         opencv_onnx::TensorProto tensor_proto = constant_node->attribute(0).t();
-        float scaleW = getMatFromTensor(tensor_proto).at<float>(0);
+        Mat scaleW = getMatFromTensor(tensor_proto);
+        CV_Assert(scaleW.total() == 1);
+        scaleW.convertTo(scaleW, CV_32F);
 
         constant_node = inputs[2].dynamicCast<ONNXNodeWrapper>()->node;
         tensor_proto = constant_node->attribute(0).t();
-        float scaleH = getMatFromTensor(tensor_proto).at<float>(0);
+        Mat scaleH = getMatFromTensor(tensor_proto);
+        CV_Assert(scaleH.total() == 1);
+        scaleH.convertTo(scaleH, CV_32F);
```

Comment:
It was needed. graph created by Pytorch 1.2 has scales as int64 datatypes whereas Pytorch 1.4 has scales as float32.

Reply:
@dkurt , these changes are needed to complete PR #16709 , so should I wait for this PR to get merge then I will rebase  or I should add these changes to PR #16709 ?